java/adapter/jdbc/src/test/java/org/apache/arrow/adapter/jdbc/JdbcToArrowTestHelper.java

```diff
@@ -378,4 +393,25 @@ public static String[] getValues(String[] values, String dataType) {
     }
     return value.split(",");
   }
+
+  public static Integer[][] getListValues(String[] values, String dataType) {
+    String[] dataArr = getValues(values, dataType);
+    Integer[][] valueArr = new Integer[dataArr.length][];
+    int i = 0;
+    for (String data : dataArr) {
+      if ("null".equals(data.trim())) {
+        valueArr[i++] = null;
+      } else if ("()".equals(data.trim())) {
+        valueArr[i++] = new Integer[0];
+      } else {
+        String[] row = data.replace("(", "").replace(")", "").split(";");
+        Integer[] arr = new Integer[row.length];
+        for (int j = 0; j < arr.length; j++) {
+          arr[j] = "null".equals(row[j]) ? null : Integer.parseInt(row[j]);
+        }
+        valueArr[i++] = arr;
+      }
+    }
+    return valueArr;
+  }
```

Comment:
It feels like this could be reused from Table.getListValues? Especially as this seems to also handle nulls